bitnami/harbor/templates/chartmuseum/chartmuseum-cm.yaml

```diff
@@ -0,0 +1,110 @@
+{{- if .Values.chartmuseum.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ template "harbor.chartmuseum" . }}-envvars"
+  labels:
+    {{ include "harbor.labels" . | nindent 4 }}
```

Comment:
Add everything on a single line:
```diff
-  labels:
-    {{ include "harbor.labels" . | nindent 4 }}
+  labels:  {{ include "harbor.labels" . | nindent 4 }}
```

Reply:
This also apply to labels at `chartmusem-dpl.yaml`, `chartmusem-pvc.yaml`, `chartmusem-secret.yaml` and `chartmuseum-svc.yaml`

Reply:
Shouldn't you add the label below too?
```
app.kubernetes.io/component: chartmuseum
```